coding-concepts.js

```diff
@@ -62,7 +62,7 @@ class Learn {
   }
 }
 const learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
\ No newline at end of file
+// a) Your answer: Learn { student: 'George', cohort: 'Bravo', year: 2023 }
+// b) Verify and explain: The variable "learnStudent" deconstructs the object "Learn". Console.log then invokes the object and passes "George" as an argurment for the name.
```

Comment:
Learn is a class and is being used to create a new object with the value George passed to the student key.